app/models/seed_data.rb

```diff
@@ -56,7 +56,7 @@ def initialize
         address_line_1: Faker::Address.street_address,
```

Comment:
This needs to be a real address

Reply:
🤷  These are not being used at the moment, I just noticed that the postcodes were not valid and therefore the data not seeded.